build.py

```diff
@@ -340,6 +331,10 @@ def build_dpdk():
     if not os.path.exists('%s/build' % DPDK_DIR):
         configure_dpdk()
 
+    # patch bpf_validate as it conflicts with libpcap
+    cmd('patch -d %s -N -p1 < %s/bpf_validate.patch || true' % (DPDK_DIR, DEPS_DIR),
```

Comment:
To be really right we should check whether it's already patched.  If we don't remove the `.orig` after patching, that could be our signal, perhaps...

Reply:
It's been a while since I wrote this :) but I think I did this so that in the future if patch (originally proposed by someone at IBM) was accepted upstream then we wouldn't fail to build.